samples/csharp/events/Program.cs

```diff
@@ -62,13 +62,13 @@ internal SearchDirectoryArgs(string dir, int totalDirs, int completedDirs)
     }
     public class FileSearcher
     {
-        public event EventHandler<FileFoundArgs> OnFoundFile;
-        internal event EventHandler<SearchDirectoryArgs> OnChangeDirectory
+        public event EventHandler<FileFoundArgs> FoundFile;
```

Comment:
Even if it's not written, but a de facto guideline, I'll make the change. Waiting for @terrajobst to weigh in.